packages/db/src/executions.ts

```diff
@@ -0,0 +1,71 @@
+import Knex from 'knex';
+
+import { ExecutionRecord } from '@cumulus/types/api/executions';
+import { PostgresExecution } from './types';
+// TODO move to common location
+const {
+  getParentExecutionCumulusId,
+  getAsyncOperationCumulusId,
+  getCollectionCumulusId,
+} = require('../../api/lambdas/sf-event-sqs-to-db-records/utils');
+
+/**
+ * Translate execution record from Dynamo to RDS.
+ *
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} dynamoRecord
+ *   Source record from DynamoDB
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} knex
+ *   Knex client
+ * @returns {PostgresExecutionRecord} - converted Execution
+ */
+export const translateApiExecutionToPostgresExecution = async (
+  dynamoRecord: ExecutionRecord,
+  knex: Knex
+): Promise<PostgresExecution> => {
+  // Map old record to new schema.
+  const translatedRecord: PostgresExecution = {
+    async_operation_cumulus_id: (
+      dynamoRecord.asyncOperationId ? await getAsyncOperationCumulusId(
+        dynamoRecord.asyncOperationId, knex
+      ) : undefined
+    ),
+    status: dynamoRecord.status,
+    tasks: JSON.stringify(dynamoRecord.tasks),
+    error: JSON.stringify(dynamoRecord.error),
+    arn: dynamoRecord.arn,
+    duration: dynamoRecord.duration,
+    original_payload: JSON.stringify(dynamoRecord.originalPayload),
+    final_payload: JSON.stringify(dynamoRecord.finalPayload),
+    workflow_name: dynamoRecord.name,
+  };
```

Comment:
I don't have a migration for `cumulus_version` here. It looks like we don't have that field in Dynamo and, if I remember correctly, this field will just be `null` if the record was created outside of PG (and migrated using this logic). Is that correct?

Reply:
I think this actually is on Dynamo as `cumulusVersion: { type: 'string' },`. Found it in schemas.js. Previously I was looking in these two places for the dynamo fields:

- The types/api/executions.d.ts Dynamo model
- The pre-existing execution test data in fakeExecutionFactoryV2

The schemas.js has this additional field that I will add to the translation here. I'll also double-check the others as I wasn't using schema.js.

Reply:
Corrected in the latest commit. I also double-checked the other fields against shema.js and updated `workflow_name` to set its value from `type` rather than `name`.  The schema for `type` is:
```
type: {
      title: 'The workflow name, e.g. IngestGranule',
      type: 'string',
    },
```